gossip/src/gossip_service.rs

```diff
@@ -229,7 +238,8 @@ pub fn get_multi_client(
     .unwrap();
     let num_nodes = tpu_addrs.len();
     (
-        ThinClient::new_from_addrs(rpc_addrs, tpu_addrs, transactions_socket),
+        //todo: make it configurable whether to use quic
```

Comment:
should just be a matter of parameterizing these functions and choosing at instantiation.  (also moving them out of gossip service?  i never understood why they lived here instead of the thin client module)

typically prefer github issues to `TODO:` as per https://github.com/solana-labs/solana/blob/fb974489a54e32eeff69c8cd4f48459e5910524a/ci/nits.sh#L64

Reply:
Changed the todo to TODO. I think this can be done in a followup - I was planning a followup PR to allow to choose the transport protocol via a command line arg.

Reply:
ah I was suggesting to just delete the todo-comment and open an issue on github in its place, but I see that I referenced the wrong comment in `nits.sh` :sweat_smile: 

we still let carl add them, so one more won't hurt. for future reference, though!